pkg/fission-cli/flag/key/key.go

```diff
@@ -38,6 +38,8 @@ const (
 	NamespacePackage     = "pkgNamespace"
 	NamespaceTrigger     = "triggerNamespace"
 	NamespaceCanary      = "canaryNamespace"
+	// using "namespaces" because "namespace" already used
+	Namespace = "namespaces"
```

Comment:
Can we define this as a "namespace" only? 

I do see we are using another flag in kubewatch CLI. We can keep both `KwNamespace` and `Namespace` flag.
And use Namespace for all our purpose.

```
-> % ack KwNamespace
pkg/fission-cli/cmd/kubewatch/create.go
64:	namespace := input.String(flagkey.KwNamespace)

pkg/fission-cli/cmd/kubewatch/command.go
34:		Optional: []flag.Flag{flag.KwName, flag.KwObjType, flag.KwNamespace, flag.NamespaceFunction, flag.SpecSave, flag.SpecDry},

pkg/fission-cli/flag/flag.go
189:	KwNamespace = Flag{Type: String, Name: flagkey.KwNamespace, Aliases: []string{"ns"}, Usage: "Namespace of resource to watch", DefaultValue: metav1.NamespaceDefault}

pkg/fission-cli/flag/key/key.go
139:	KwNamespace = "namespace"
```